src/python/WMComponent/WorkflowUpdater/WorkflowUpdaterPoller.py

```diff
@@ -250,6 +251,7 @@ def __init__(self, config):
 
         # parse mandatory attributes from the configuration
         self.config = config
+        self.dbsUrl = getattr(config.WorkflowUpdater, "dbsUrl", "https://cmsweb-prod.cern.ch/dbs/prod/global/DBSReader")
```

Comment:
Did we already have had another thread on how to add those extra config values to the WMAgent configuration file. I'll need it in order to have them persistent in the agent.

Maybe we also need to update the WMAgent config templates: https://github.com/dmwm/WMCore/blob/master/etc/WMAgentConfig.py?